core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts

```diff
@@ -480,8 +480,14 @@ export class MxGraphVisitorHelper {
       const aspectVersionedNamespace = rdfModel.getAspectModelUrn().replace('#', '');
       const [elementVersionedNamespace] = modelElement.aspectModelUrn.split('#');
 
+      let metaModelVersion =  modelElement.metaModelVersion;
+      if(modelElement instanceof DefaultEntityInstance){
+        metaModelVersion = modelElement.entity.metaModelVersion;
+      }
+
       return {
-        version: modelElement.metaModelVersion,
+        version: this.extractText(rdfModel.absoluteAspectModelFileName,':',':'),
```

Comment:
Have a look on rdf-model-utils.ts, there are some static methods that you can use to extract the version.

Reply:
done